frontend/src/metabase/forms/components/FormDateInput/FormDateInput.tsx

```diff
@@ -50,14 +50,14 @@ export const FormDateInput = forwardRef(function FormDateInput(
   return (
     <DateInput
       fw="bold"
+      onChange={handleChange}
```

Comment:
we need to be able to override this onChange to be able to set the date and time separately